README.md

```diff
@@ -236,13 +238,14 @@ image references, you can specify this in the `helm install` command:
 ```
 helm --namespace injectors \
  install \
+ --set "deploymentApiVersion=apps/v1" \
  --set "caBundle=$(kubectl -n kube-system \
     get configmap \
     extension-apiserver-authentication \
     -o=jsonpath='{.data.client-ca-file}' \
   )" \
- --set secretless-image=path/to/secretless/container/image/repo/and/tag" \
- --set authenticator-image=path/to/authenticator/container/image/repo/and/tag" \
+ --set secretlessImage=path/to/secretless/container/image/repo/and/tag" \
+ --set authenticatorImage=path/to/authenticator/container/image/repo/and/tag" \
```

Comment:
is this fixing a typo in the docs or are we making a backwards-incompatible change?

looks like the former, but figured it'd be good to verify

Reply:
Definitely fixing a typo